pysat/tests/classes/cls_instrument_access.py

```diff
@@ -1008,28 +1008,3 @@ def test_basic_variable_renaming(self, lowercase, mapper):
             assert key not in self.testInst.variables
             assert key not in self.testInst.meta.keys()
         return
-
-    def test_generic_meta_translator(self):
```

Comment:
I can't see why this test was removed?

Reply:
This test validates that the deprecated function behaves identical to the replacement function. See Line 0133